_config.yml

```diff
@@ -44,6 +44,13 @@ defaults:
     values:
       layout: "tour"
       permalink: "/tour/:basename/"
+  -
+    scope:
+      path: "tour-jshell"
+      type: "pages"
+    values:
+      layout: "tour-jshell"
+      permalink: "/tour-jshell/:basename/"
```

Comment:
This should replace the existing tour. Not sit adjacent to it.

Reply:
If we replace the existing tour then users using any 1.x or 2.0.x version would not be able to take advantage of the tour. I would think at least for a while it would be best to keep both accessible.

Reply:
JShell can be used with the older version. We just don't have the convenience tools to automatically set up the class path. The jshell-based tour would still be applicable for those versions, but setting up the classpath could be left as an exercise to the user.

Also, the tour targets new users. New users are not typically starting off with old versions.

I would also call the tour "Interactive" rather than "JShell". JShell is the just the tool, but the point of the tour using JShell is to be more interactive, so that's the characteristic I would emphasize in any menus/navigation/page titles.